docs/usage/configuration-options.md

```diff
@@ -98,6 +98,16 @@ Currently it is needed/supported for the `helm-requirements` manager only.
 }
 ```
 
+When using a `docker` proxy cache `aliases` can be used to map the images prefix on the proxy to the original prefix:
```

Comment:
```suggestion
The following config will result in Renovate treating images starting with `registry.example.com/proxy-cache` as if the image starts with `docker.io` instead.
Renovate will get the tag list from Docker Hub. which helps with/is good for (INSERT EXPLANATION HERE).

When using a `docker` proxy cache `aliases` can be used to map the images prefix on the proxy to the original prefix:
```

We're currently missing a explanation on why you might want this,, can you maybe add one?
For an outsider (like me) it's not clear from the current text why I might need this. 😄 

I think the problem is related to this quote?

> Since these proxies do not return a full tag list, renovate is unable to update images containing the prefix.